requirements.txt

```diff
@@ -1,2 +1,4 @@
+wheel==0.34.2
```

Comment:
Are we sure that the `wheel` package is required?

Reply:
I remember needing it to install neo4j. At least the method I used.

Reply:
Np, just wanted to make sure